src/api/privateApi.js

```diff
@@ -264,8 +264,7 @@ function handleSendRequestFailures(requestMethod, requestType, options, error) {
 	});
 }
 
-
-module.exports = {
+export default {
```

Comment:
Maybe it makes more sense to export the functions individually than to export an object? That way we can also do nice destructuring directly when importing. We already have this pattern in `src/transactions/utils.js`.